_conferences/fullstack-flutter-may-2024.md

```diff
@@ -4,11 +4,11 @@ website: https://fullstackflutter.dev/
 location: Online
 online: true
 
-date_start: 2024-05-14
-date_end:   2024-05-14
+date_start: 2024-05-28
+date_end:   2024-05-28
```

Comment:
They moved it because it collided with Google I/O right?

Reply:
probably